### DISCUSSION TITLE
foobar2000

### DISCUSSION THREAD

**Discussion by rollingmoai** [catppuccin/catppuccin#1834] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
No

**Additional context 📷 (optional)**
@niivu's newly released [Catppuccin theme for Windows](https://www.deviantart.com/niivu/art/Catppuccin-for-Windows-908988670) contains a foobar2000 skin that we can use.

**Comment by kingofcube** [comment]
please
